### PR TITLE
Fix `generate_context.sh`

### PR DESCRIPTION
**Pull request by ribbanya**

### CODE REVIEW

File: src/common_structs.h [hunk 1]
```diff
@@ -126,4 +128,40 @@ typedef enum GroundOrAir {
...
+    uint io_mode : 3;
+    uint buffer_mode : 2;
+    uint file_kind : 3;
+    uint file_orientation : 2;
+    uint binary_io : 1;
+} __file_modes;
+
+typedef struct FILE FILE;
+
+struct FILE {
```
Comment (r-burns) [inline src/common_structs.h:142]: This should be in stdio.h

File: src/melee/ft/types.h [hunk 2]
```diff
@@ -492,7 +492,8 @@ typedef enum ftCommonAction {
     ASID_CAPTUREDAMAGECRAZYHAND,
     ASID_CAPTUREWAITCRAZYHAND,
     ASID_THROWNCRAZYHAND,
-    ASID_BARREL
+    ASID_BARREL,
+    ASID_MAX
```
Comment (r-burns) [inline src/melee/ft/types.h:496]: Maybe ASID_COUNT or similar is more accurate here? To me ASID_MAX implies the max valid ftCommonAction value, or ASID_MAX == ASID_BARREL.

Comment (ribbanya) [reply src/melee/ft/types.h:496]: I was just following `FTKIND_MAX` but I had planned on changing them all to pascal case and `max` to `count` anyway. Just wanted the context to compile in this case.

Comment (r-burns) [reply src/melee/ft/types.h:496]: Ah didn't realize there was precedent here, sounds good.

**Review by ribbanya — COMMENTED** [review]
This seems fine to merge for now, but I want to get rid of `generate_context.sh` soon in favor of a pure Python solution.

**Review by r-burns — APPROVED** [review]

**Pull request merged**